### DISCUSSION TITLE
Question: How do I quickly put a cursor on 2 or more lines?

### DISCUSSION THREAD

**Discussion by James-Firth** [helix-editor/helix#9516] [Q&A] [answered]
I checked the docs and tutor but nothing stood out to me for this beyond memorizing a regex (maybe)

---

Let's say I have the text

```
#################
# my cool ascii
# art
########################
```

and I want to trim both lines so I can align it to the text, so naturally I want to start deleting on both and then
only the bottom.

How do I get multiple cursors to show up either like of the examples below, quickly and easily?
My first thought was I could select and jump but that didn't work. And most examples shown involve adjacent lines. I'm wondering about non-adjacent lines without futzing with regexes.

```
#################█ 
# my cool ascii
# art
########################█ 
```
or
```
█#################
# my cool ascii
# art
█######################## 
```

Thanks in advance and if this isn't a feature, maybe it should be? (or not!)

**Comment by grv07** [comment]
Pls open an issue for that one, I don't think it is supported but Helix.
It would be nice if you add any example editor that support this.

**Reply by James-Firth** [reply]
A reply the day after you gave an answer for this particular example. It would be cool to be able to jump to arbitrary, non-adjacent lines with text in between, but I'll have to see if that particular need pops up before I want to open an issue.

**Comment by the-mikedavis** [comment, marked as answer]
If your cursor is on the last `#` on the first line and you use `C` it will create a cursor on the next suitable line which will be the last line. Then you can use `gl` to move both cursors to the ends of their lines

**Reply by James-Firth** [reply]
Ah! I hadn't thought about the fact it would notice no characters in the columns below until the "next suitable line" like that, interesting!

I'll have to think/see if I come across a usecase where there's a need to put cursors on arbitrary lines (that have text or whitespace in the way where this wouldn't work). If I think of one maybe I'll open an issue.

Thanks I'll mark this as an answer since it does solve that initial problem!
